selectors/emojis.ts

```diff
@@ -54,10 +54,7 @@ export function isCustomEmojiEnabled(state: GlobalState) {
 export const getOneClickReactionEmojis = (state: GlobalState) => {
     const recentEmojis = getRecentEmojis(state).slice(-3);
     const emojiMap = getEmojiMap(state);
-    const emojis = [];
-    for (let i = 0; i < recentEmojis.length; i++) {
-        emojis.push(emojiMap.get(recentEmojis[i]));
-    }
 
-    return emojis.reverse();
+    const emojis = recentEmojis.slice(-3).map((recentEmoji) => emojiMap.get(recentEmoji)).filter(Boolean).reverse();
```

Comment:
Did you mean to slice this a second time?

Also, can you convert this whole thing into a memoized selector? It's returning a new array each time currently which'll likely affect performance

Reply:
- Removed the double slice
- So i had converted this to memoized selector in the start, but it wont fire again when there is change in the recent emoji select as it has to get the new from the local storage

Reply:
Shouldn't `getRecentEmojis` return a new result in that case though? Maybe I'm missing something because I know the storage code doesn't always work the way it should